tests/test_conn.py

```diff
@@ -19,6 +19,22 @@ def test_token_twice(conn):
         conn.add_token("burner2", expiry="1w", token="1w_7wDioPeeXyZx96v3", prefix="xp")
 
 
+def test_add_del_user(conn, mailcow):
+    token = conn.add_token("pytest:burner1", expiry="1w", token="1w_7wDioPeeXyZx96v3", prefix="pp")
+    usecount = conn.get_tokeninfo_by_name(token.name).usecount
+    addr = conn.add_email_account_tries(token, tries=10).addr
+    assert usecount + 1 == conn.get_tokeninfo_by_name(token.name).usecount
+
+    assert mailcow.get_user(addr)
+    assert conn.get_user_by_addr(addr)
+
+    conn.delete_email_account(addr)
+
+    with pytest.raises(TypeError):
+        conn.get_user_by_addr(addr)
+    assert not mailcow.get_user(addr)
+
+
```

Comment:
tbh, this test is not super useful here. I wrote it before I found out that the bug is a web.py bug, and just kept it around because "more tests, more good" :D